### PR TITLE
Fix Readme with local setup and other relevant information

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Fyle QuickBooks Integration</title>
```
Comment (ashwin1111) [inline src/index.html:5]: Fyle Xero Integration

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//     RedirectTo: 'workspaces',
+//     PathMatch: 'full'
+//   }
+// ];
+
+// @NgModule({
+//   Imports: [RouterModule.forRoot(routes)],
+//   Exports: [RouterModule]
+// })
+// Export class AppRoutingModule { }
```
Comment (ashwin1111) [inline src/app/app-routing.module.ts:35]: Why is this completely commented out? why not just make the routes []

File: src/app/app.component.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<!-- <router-outlet></router-outlet> -->
+<h1>Xero Integration</h1>
```
Comment (ashwin1111) [inline src/app/app.component.html:2]: Why?

File: src/app/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import { BrowserModule } from '@angular/platform-browser';
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { JwtHelperService, JWT_OPTIONS } from '@auth0/angular-jwt';
+import { MatSnackBarModule, MAT_SNACK_BAR_DEFAULT_OPTIONS } from '@angular/material/snack-bar';
+
+import * as Sentry from '@sentry/angular';
+
+// Import { AppRoutingModule } from './app-routing.module';
```
Comment (ashwin1111) [inline src/app/app.module.ts:11]: same

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: src/app/app.component.html (hunk 3 above)
Comment (DhaaraniCIT) [reply src/app/app.component.html:2]: just for a placeholder

Comment (ashwin1111) [reply src/app/app.component.html:2]: not needed, please remove, it doesn't add any value

**Review by ashwin1111 — APPROVED** [review]

File: src/app/app.component.html (hunk 3 above)
Comment (DhaaraniCIT) [reply src/app/app.component.html:2]: shall i keep it as a blank page

Comment (ashwin1111) [reply src/app/app.component.html:2]: yes

**Pull request merged**